packages/plugins/tests/product-price-round.test.js

```diff
@@ -0,0 +1,66 @@
+import {configureProductPriceRounding, ProductPriceRoundSettings, roundToNext} from '../src/pricing/product-price-round'
+
+describe('roundToNext', () => {
```

Comment:
same here: split the tests up and call them:

roundToNext.test.js, configureProductPriceRounding.test.js